adoc/SA_vars.adoc

```diff
@@ -17,9 +18,13 @@ endif::[]
 
 // could be "Azure","AWS","GCE" or "libvirt"
 //:cloud: Azure
-:cloud: AWS
+//:cloud: AWS
+:cloud: GCP
 :sles: SUSE Linux Enterprise Server
 :sles4sap: {sles} for SAP Applications
 :gcloud: Google Cloud
+:proj_ver: 6.0.0
```

Comment:
i dont think we should use different project versions - as it would maybe change too much.
Btw. the master branch is updated with the fixes in the 6.0.x versions - so master is the way to go

If we go this route - we need to push out a new document with every change in the project.

We should mention something like "we used v6xx as the time of writing"...

Reply:
I have to stick with v 6.0.3 for GCP as it contains several fixes and enhancements reviews.